### ISSUE TITLE
hi，yjh，This article is very useful to me，could you tell me the improvements of YOWO ? use the new 3D-branch?

### ISSUE THREAD

**Issue by xxz1522** [yjh0410/PyTorch_YOWO#1]

**Comment by yjh0410** [comment]
@xxz1522  Our implementation of YOWO is identical to the official YOWO in terms of network structure. Even our Yow-Nano is essentially replacing 3D-ResNext-101 with 3D-ShuffleNetv2, provided by the YOWO‘s author team. 

The reason why we achieved a better YOWO, I think may be due to the following convenience:

1. **Better label assignment**. YOWO adopts the original YOLOv2 label assignment, that is, only the prediction box whose IoU is the largest will be matched with the groundtruth. However, our YOWO matches anchor boxes with IoU over 0.5 to groundtruth. In our YOWO, a groundtruth may match multiple positive samples.

2. Better loss. Our confidence loss and classification loss are the same as the official YOWO, but our regression loss adopts GIoU, therefore, the  box regression is more stable. Moreover, the sum of our losses is normalized by the batch size, which is conducive to stable training.

3. Better post-processing.

**Comment by xxz1522** [comment]
Thanks for your answer! Is there a paper about this code? If there is , could you tell me the name of the paper?

**Comment by yjh0410** [comment]
@xxz1522 I haven't written a paper about our reimplemented YOWO yet. I have this plan to write a technical report which will be post on Arxiv. I'll contact you once I finish writing that technical report.

**Comment by xxz1522** [comment]
Thank you!

**Comment by yjh0410** [comment]
@xxz1522 
Hi, dear friend. I recently published a paper about my reimplemented YOWO on the Arxiv. The title of my paper is YOWO-Plus: An Incremental Improvement. You can read the paper from [here](https://arxiv.53yu.com/pdf/2210.11219).
